subiquity/ui/views/filesystem/partition.py

```diff
@@ -405,6 +406,9 @@ def __init__(self, parent, disk, partition=None):
                 lvm_names.remove(partition.name)
         else:
             initial['fstype'] = 'ext4'
+            if gap:
+                initial['size'] = humanize_size(self.gap.size)
```

Comment:
This is a change in behaviour I think? size used to be empty by default

Reply:
Dropped